src/test/java/teammates/test/pageobjects/IndexPage.java

```diff
@@ -0,0 +1,15 @@
+package teammates.test.pageobjects;
+
+public class IndexPage extends AppPage {
+
+    public IndexPage(Browser browser) {
+        super(browser);
+    }
+
+    @Override
+    protected boolean containsExpectedPageContents() {
+        String pageSource = getPageSource();
+        return pageSource.contains("Student Login") && pageSource.contains("Instructor Login");
```

Comment:
Changed to looking for this (pretty much because all other `containsExpectedPageContents()` also look for the main heading):
![image](https://cloud.githubusercontent.com/assets/7261051/11557169/5bc24c10-99e6-11e5-847a-6ba148c0f923.png)